n2y/notion_mocks.py

```diff
@@ -144,3 +144,47 @@ def mock_select_option(name, **kwargs):
 
 def mock_relation_value():
     return {"id": mock_id()}
+
+
+def mock_page(title="Mock Title"):
+
+    # Returns a mock version the data that would be returned by
+    # the Notion API on a client.get_page() request.
+
+    user = mock_user()
+    created_time = datetime.now().isoformat()
+    notion_id = mock_id()
+    hyphenated_title = title.replace(" ", "-")
+
+    page_object = {
+        'object': 'page',
+        'id': notion_id,
+        'created_time': created_time,
+        'last_edited_time': created_time,
+        'created_by': user,
+        'last_edited_by': user,
+        'cover': None,
+        'icon': None,
+        'parent': {'type': 'page_id', 'page_id': mock_id()},
+        'archived': False,
+        'properties': {
+            'title': {
```

Comment:
I think you can simplify this section by using `mock_rich_text_array`. Grep for that function to see examples of how it's 'used.

Reply:
Yes.  I see that.  Thanks for the tip.

Reply:
Is there an instance foreseen in which one might want to use mock_annotations() outside of the mock_rich_text() call?

Reply:
@idolcemia I don't recall offhand, but I _think_ there was a case for this. If you grep for `mock_annotations` I bet you'll find it. If not, I can't think of any.